niceman/interface/create.py

```diff
@@ -172,13 +144,8 @@ def __call__(resource, resource_type, config, resource_id, clone, only_env,
         env_resource = ResourceManager.factory(config)
 
         # Set resource properties to any backend specific command line arguments.
-        for backend_arg in backend:
-            key, value = backend_arg.split("=")
-            if hasattr(env_resource, key):
-                config[key] = value
-                setattr(env_resource, key, value)
-            else:
-                raise NotImplementedError("Bad --backend paramenter '{}'".format(key))
+        if backend:
+            config = backend_set_config(backend, env_resource, config)
```

Comment:
imho better to have a function without 'side effect' of modifying config in place and then also returning it...  what if 

`backend_set_config` -> `get_backend_config(backend, env_resource)` which returns a backend_config, and then you just do  `config.update(backend_config)`, i.e. update with those new settings.  IMHO makes it clearer